### PR TITLE
🛀 A lot less USSD

### PR DESCRIPTION
**Pull request by rowanseymour**
Removes USSD functionality, leaving only:

- The USSDSession model itself
- USSD functionality in the old editor.. no point in even messing with that right now

### CODE REVIEW

File: temba/contacts/tests.py [hunk 1]
```diff
@@ -3392,7 +3392,7 @@ def test_history(self):
             )
 
             # fetch our contact history
-            with self.assertNumQueries(70):
+            with self.assertNumQueries(69):
```
Comment (rowanseymour) [inline temba/contacts/tests.py]: no more checking if we have a USSD channel in templates

File: temba/api/v2/serializers.py [hunk 2]
```diff
@@ -716,12 +716,7 @@ def save(self):
 
 
 class FlowReadSerializer(ReadSerializer):
-    FLOW_TYPES = {
-        Flow.TYPE_MESSAGE: "message",
-        Flow.TYPE_VOICE: "voice",
-        Flow.TYPE_USSD: "ussd",
-        Flow.TYPE_SURVEY: "survey",
-    }
+    FLOW_TYPES = {Flow.TYPE_MESSAGE: "message", Flow.TYPE_VOICE: "voice", Flow.TYPE_SURVEY: "survey"}
```
Comment (rowanseymour) [inline temba/api/v2/serializers.py]: since #2259 all USSD flows are inactive

File: templates/flows/flow_editor.haml [hunk 3]
```diff
@@ -59,9 +59,9 @@
     window.uploadMediaURL = '{% url "flows.flow_upload_media_action" object.pk %}';
     window.mutable = {{mutable|lower}};
     window.ivr = '{{object.flow_type}}' == 'V';
-    window.ussd = '{{object.flow_type}}' == 'U';
-    window.useMailroom = {{has_mailroom|lower}} && '{{object.flow_type}}' != 'U';
-    window.has_ussd_channel = {{has_ussd_channel|lower}};
+    window.ussd = false;
+    window.useMailroom = {{has_mailroom|lower}};
+    window.has_ussd_channel = false;
```
Comment (rowanseymour) [inline templates/flows/flow_editor.haml]: keeping these vars but setting them to false to avoid messing with the old editor code

**Comment by dodobas** [comment]
Would it be possible to squash commits, there are far to many commits that only have WIP as a commit message ... far to many in this case is any number that is not 0 (zero)

**Comment by dodobas** [comment]
```
git log --pretty=format:"%an : %s" | grep -i WIP| cut -d ':' -f 1| sort | uniq -c
      5 Eric Newcomer 
      2 ewheeler 
      5 Fekete Gyorgy 
      1 jofomah 
     35 Nic Pottier 
      4 Norbert Kwizera 
     50 Rowan Seymour
```

**Comment by rowanseymour** [comment]
I've probably done more large refactoring PRs than most on RP. We can have discussion about git best practices (I'm a liberal committer.. if I leave for the day I commit) but I'm not inclined to squash a PR into one commit after the fact.

**Comment by nicpottier** [comment]
As a rule I do not believe in squashing commits, you are erasing history. There are plenty of ways of viewing the simplified git (such as this very website) so there's no real incentive to squash.

**Pull request merged**